mediathread/djangosherd/models.py

```diff
@@ -415,14 +418,16 @@ class DiscussionIndex(models.Model):
     2. what discussions are connected to a particular asset (for asset page)
     3. what new comments should be featured in Clumper
     """
-    participant = models.ForeignKey(User, null=True)
-    collaboration = models.ForeignKey(Collaboration)
+    participant = models.ForeignKey(User, null=True, on_delete=models.CASCADE)
+    collaboration = models.ForeignKey(Collaboration, on_delete=models.CASCADE)
 
-    asset = models.ForeignKey(Asset, null=True,
-                              related_name="discussion_references")
+    asset = models.ForeignKey(
+        Asset, null=True,
+        related_name="discussion_references",
+        on_delete=models.CASCADE)
 
     # just for use-case #3
-    comment = models.ForeignKey(Comment, null=True)
+    comment = models.ForeignKey(Comment, null=True, on_delete=models.PROTECT)
```

Comment:
Also wondering about PROTECT here...If a comment is deleted, it makes sense to me that the DiscussionIndex would also be deleted.

Reply:
Yeah, I agree. I'll update that.